src/components/ChannelList.vue

```diff
@@ -1,46 +1,61 @@
 <template>
-    <div class="kiwi-channellist">
-        <div class="kiwi-channellist-nav">
-            <a
-                :class="{
-                    'u-button-primary': !isLoading,
-                    'u-button-secondary': isLoading,
-                }"
-                class="u-button kiwi-channellist-refresh"
-                @click="maybeUpdateList">
-                <i v-if="!isLoading" class="fa fa-refresh" aria-hidden="true"/>
-                <i v-else class="fa fa-refresh fa-spin" aria-hidden="true"/>
-            </a>
-
-            <div class="kiwi-channellist-pagination">
-                <a @click="prevPage"><i class="fa fa-step-backward" aria-hidden="true"/></a>
-                {{ page + 1 }} / {{ maxPages + 1 }}
-                <a @click="nextPage"><i class="fa fa-step-forward" aria-hidden="true"/></a>
+    <div :class="{'kiwi-channellist-padding-top': !list.length}" class="kiwi-channellist">
+        <div class="kiwi-channellist-content-container">
+            <div class="kiwi-channellist-nav">
+                <form class="u-form kiwi-channellist-search" @submit.prevent>
+                    <input v-model="search" :placeholder="$t('do_search')" class="u-input" >
+                    <a
+                        :class="{
```

Comment:
put class on the same line as a, lint will then make you line everything under :class, should make it a little neater

Reply:
i say this then look into StateBrowserNetwork.vue and its all done your way. so maybe ignore me